src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: Current session ID 
+type: apiDoc
+shortDescription: 'Returns ID for current session.'
+tags:
+  - Mobile monitoring
+  - Mobile SDK API
+  - Custom instrumentation
+metaDescription: Mobile app monitoring API used to return the ID for current session.
+redirects:
+- /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/currentsessionid-ios-sdk-api
+- /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/currentsessionid-android-agent-api
+- /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/current-session-id
+- /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/current-session-id
+- /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/currentsessionid-android-sdk-api
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+        <TabsBarItem id="capacitor">
+            Capacitor
+        </TabsBarItem>
+        <TabsBarItem id="cordova">
+            Cordova
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React Native
+        </TabsBarItem>
+        <TabsBarItem id="xamarin">
+            Xamarin
+        </TabsBarItem>
+        <TabsBarItem id="maui">
+            .NET MAUI
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java] 
+
+```java
+NewRelic.currentSessionId()
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+NewRelic.currentSessionId()
+```
+
+## Description [#description]
+
+Returns ID for current session.
+
+This method is useful for consolidating monitoring of app data (not just New Relic data) based on a single session definition and identifier. For example, you might want to use the same identifier for marketing analytics, or user analytics.
+
+For more context on how to use this API, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/work-android-sdk-api#attributes-events-insights).
+
+Compatible with all agent versions.
+
+## Return values [#return-values]
+
+Returns ID string for the current session.
+
+## Example [#example]
+
+### Java [#java]
+
+```java
+String sessionId = NewRelic.currentSessionId();
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
```suggestion
```kotlin
        val sessionId = NewRelic.currentSessionId()
```
```